metron-analytics/metron-profiler-client/src/main/java/org/apache/metron/profiler/client/stellar/GetProfile.java

```diff
@@ -132,36 +157,26 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(GetProfile.class);
 
-  /**
-   * A client that can retrieve profile values.
-   */
+  // Cached client that can retrieve profile values.
```

Comment:
Just curious, why switch the comment style here?  When should I use javadoc comments versus regular comments?